frontend/src/app/init-vendors.js

```diff
@@ -82,5 +85,3 @@ require('expose-loader?URI!URIjs');
 require('URIjs/src/URITemplate');
 
 require("expose-loader?I18n!../vendor/i18n");
-
-require("fullcalendar/dist/locale-all.js");
```

Comment:
Is the goal then to split up the init-vendors into module specific inits in the long run?